teensytoany/teensytoany.py

```diff
@@ -301,7 +308,17 @@ def i2c_write_payload(self, address: int, payload: Sequence) -> None:
                 f"0x{address:02x} 0x{register_address:04x} 0x{data:04x}")
         else:
             raise NotImplementedError()
+        """
+
+    def i2c_read_payload(self, address: int, register_address: int, num_bytes: int) -> Sequence:
+
+        cmd = f"i2c_read_payload 0x{address:02x} 0x{register_address:02x} {num_bytes}"
+        returned = self._ask(cmd)
+        register_data = [int(val, base=0) for val in returned.split()] # returns big endian
```

Comment:
I'm not sure what returns big endian means. Big endian is usually reserved for data that is larger than 8bits. When data is 8bits long, I don't think "endian" has any meaning.

Reply:
It just means if a user is asking for example bytes 0x00 and 0x01, it will return the values in the order [0x00, 0x01] instead of [0x01, 0x00]. So it will return low bytes at the front of a Sequence.

Reply:
hmm ok. i guess this is what i would expect from byte addressable protocols. I2C is inherently byte by byte. But ok.